src/Cosserat/constraint/CosseratActuatorConstraint.cpp

```diff
@@ -28,36 +28,22 @@
 *                                                                             *
 ******************************************************************************/
 
-
-#include "CosseratActuatorConstraint.inl"
+#include <Cosserat/constraint/CosseratActuatorConstraint.inl>
 #include <Cosserat/config.h>
 #include <sofa/core/ObjectFactory.h>
 
-namespace sofa::component::constraintset
+template class SOFA_COSSERAT_API sofa::component::constraintset::CosseratActuatorConstraint<sofa::defaulttype::Vec3Types>;
```

Comment:
What is the use of having this explicit template instantiation without  adding the external instantiation in the header ?

Reply:
I am not sure. I think the external instantiation must be done in the header anyway. I just uniformized the explicit template instantiations for all components, but only in the cpp files.

Reply:
Ok, would require another PR for this ! Then it is all good to me !